data/json/artifact/altered_object_active.json

```diff
@@ -0,0 +1,116 @@
+[
+  {
+    "type": "SPELL",
+    "id": "AO_FRIENDLY_VORTEX",
+    "name": { "str": "Artifact Vortex" },
+    "description": "Summon Vortexes",
+    "valid_targets": [ "ground" ],
+    "effect": "summon",
+    "effect_str": "mon_vortex",
+    "base_casting_time": 100,
+    "message": "Space twists and deforms around you.",
+    "min_damage": 1,
+    "max_damage": 3,
+    "min_aoe": 5,
+    "max_aoe": 5,
+    "flags": [ "RANDOM_DAMAGE", "HOSTILE_50" ],
+    "min_duration": 40000,
+    "max_duration": 40000
+  },
+  {
+    "type": "SPELL",
+    "id": "AO_CALL_OF_TINDALOS",
+    "name": { "str": "Artifact Call of Tindalos" },
+    "description": "Summon hostile hounds of tindalos",
+    "valid_targets": [ "ground" ],
+    "effect": "summon",
+    "effect_str": "mon_hound_tindalos",
+    "base_casting_time": 100,
+    "message": "The world unravels into geometric shapes, and between the chaos you spot a horrid prowling beast.",
+    "min_damage": 1,
+    "max_damage": 1,
+    "min_aoe": 15,
+    "max_aoe": 15,
+    "min_duration": 40000,
+    "max_duration": 40000,
+    "flags": [ "HOSTILE_SUMMON" ]
+  },
+  {
+    "id": "AO_FORCE_PULL",
+    "type": "SPELL",
+    "name": "Artifact Force Pull",
+    "description": "Pull objects.",
+    "effect": "area_pull",
+    "effect_filter": [ "ally", "hostile", "item", "fd_fire", "fd_blood" ],
+    "effect_iterations": 9,
```

Comment:
What's that?

Reply:
Its from  #31783. But there are no docs related to it.

I just copied the json object from there, and checked that the spell worked in-game. Never verified what this entry does.

Reply:
the CI is saying it isn't reading it, and magic doesn't have any loading code for it.